.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Calance generic CI-CD
+on:
+  push:
+    tags:
+    - v[0-9]+.[0-9]+.[0-9]+-rc[0-9]+
+    branches:
+    - 'main'
+    paths:
+    - 'Dockerfile'
+    - '**.py'
+  workflow_dispatch:
+
+
+jobs:
+  build-docker:
```

Comment:
can't you call the `docker build` job from `calance-workflows` common workflows repository?

Reply:
yes we can, i have changed it too